index.js

```diff
@@ -1448,14 +1447,179 @@ module.exports = {
     });
   },
 
+  /********************
+   * Raw transactions *
+   ********************/
+
+  parseRawTransactionResponse: function (rawTransactionResponse, packaged, address, privateKey, cost, callback) {
+    if (this.debug.broadcast) {
+      console.debug("[ethrpc] sendRawTransaction response:", rawTransactionResponse);
+    }
+    if (!rawTransactionResponse) {
+      if (!isFunction(callback)) return errors.RAW_TRANSACTION_ERROR;
+      return callback(errors.RAW_TRANSACTION_ERROR);
+    }
+    if (rawTransactionResponse.error) {
+      if (rawTransactionResponse.message.indexOf("rlp") > -1) {
+        var err = clone(errors.RLP_ENCODING_ERROR);
+        err.bubble = rawTransactionResponse;
+        err.packaged = packaged;
+        if (!isFunction(callback)) return err;
+        return callback(err);
+      } else if (rawTransactionResponse.message.indexOf("Nonce too low") > -1) {
+        if (this.debug.broadcast || this.debug.nonce) {
+          console.info("[ethrpc] Nonce too low, incrementing:", rawTransactionResponse.message, packaged, this.rawTxMaxNonce);
+        }
+        ++this.rawTxMaxNonce;
+        delete packaged.nonce;
+        return this.setNonceThenSubmitRawTransaction(packaged, address, privateKey, callback);
+      }
+      if (!isFunction(callback)) return rawTransactionResponse;
+      return callback(rawTransactionResponse);
+    }
+
+    // rawTransactionResponse is the txhash if nothing failed immediately
+    // (even if the tx is nulled, still index the hash)
+    this.rawTxs[rawTransactionResponse] = {tx: packaged, cost: abi.unfix(cost, "string")};
```

Comment:
@MicahZoltu first of all, fix/unfix are used all throughout the codebase.  If they were incorrect, all values from the application would be wrong.  So you should probably start from the assumption that this stuff all works and you are looking for the gap in your understanding.

That hex value is a hex-encoded fixed-point number.  Like every other value we get back from the contracts it is an int256.  To show to users / to work with conveniently in JavaScript we would like it to be a normal base10 number, i.e., not fixed-point.  However we do not want to use JS floating point because binary floating point results in rounding errors.  Instead we deal with the numbers as stringified base10 numbers.

Demonstration that what I'm saying is correct:
```
> abi.fix("3.1415", "hex")
'0x2b98d99b09e3c000'
```
This is the same result we get from:
```
> new BigNumber("3.1415", 10).times(new BigNumber(10, 10).toPower(18)).toString(16)
'2b98d99b09e3c000'
```
And if we use `abi.unfix` we should get back the (stringified) float that we started with, `3.1415`:
```
> abi.unfix("0x2b98d99b09e3c000", "string")
'3.1415'
```